.drone.yml

```diff
@@ -106,6 +106,66 @@ steps:
     image: golang:1.17.5
     commands:
       - make build-all
+---
+kind: pipeline
+type: kubernetes
+name: build-and-push-images-on-push-linux
+
+trigger:
+  branch:
+    - master
```

Comment:
Will we need to build releases for `branch/v*` as well?

Reply:
I noticed that `branch/v*` is referenced in the drone branch triggers but there isn't any `branch/v*` branches. This PR https://github.com/gravitational/teleport-plugins/pull/116 added them but makes no mention as to their purpose within the repository. 

However, if desired I can add them in.